src/main/java/org/opentripplanner/standalone/config/sandbox/FlexConfig.java

```diff
@@ -5,17 +5,14 @@
 
 import java.time.Duration;
 import org.opentripplanner.standalone.config.framework.json.NodeAdapter;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 public class FlexConfig {
 
-  public static final FlexConfig DEFAULT = new FlexConfig();
-
   private static final Duration MAX_TRANSFER_DURATION = Duration.ofMinutes(5);
   private static final Duration MAX_FLEX_TRIP_DURATION = Duration.ofMinutes(45);
```

Comment:
We could just inline the values in the constructor, now that there is a `DEFAULT` instance

Reply:
Yes, that is a better solution.